File: src/Squidex/app/features/administration/pages/users/messages.ts
/*
 * Squidex Headless CMS
 *
 * @license
 * Copyright (c) Sebastian Stehle. All rights reserved
 */

export class UserCreated {
    constructor(
        public readonly id: string,
        public readonly email: string,
        public readonly displayName: string,
        public readonly pictureUrl: string
    ) {
    }
}

export class UserUpdated {
    constructor(
        public readonly id: string,
        public readonly email: string,
        public readonly displayName: string
    ) {
    }
}

File: backend/src/Squidex.Infrastructure/EventSourcing/Consume/Messages.cs
// ==========================================================================
//  Squidex Headless CMS
// ==========================================================================
//  Copyright (c) Squidex UG (haftungsbeschraenkt)
//  All rights reserved. Licensed under the MIT license.
// ==========================================================================

#pragma warning disable MA0048 // File name must match type name
#pragma warning disable SA1313 // Parameter names should begin with lower-case letter

namespace Squidex.Infrastructure.EventSourcing.Consume;

public sealed record EventConsumerReset(string Name);

public sealed record EventConsumerStart(string Name);

public sealed record EventConsumerStop(string Name);
